<launch>
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find cvg_sim_gazebo)/worlds/ardrone_testworld.world"/>
  </include>

  <include file="$(find cvg_sim_gazebo)/launch/spawn_quadrotor.launch">
    <arg name="model" value="$(find cvg_sim_gazebo)/urdf/quadrotor_sensors.urdf.xacro"/>
  </include>


  <node name="drone_stateestimation" pkg="tum_ardrone" type="drone_stateestimation">
  </node>
  <node name="drone_autopilot" pkg="tum_ardrone" type="drone_autopilot">
  </node>
  <node name="drone_gui" pkg="tum_ardrone" type="drone_gui">
  </node>


  <!--drone control-->
  <node pkg="drone_control" name="deep_navigation" type="deep_navigation_RL.py" output="screen" />

</launch>
